engine/src/main/java/org/destinationsol/game/screens/MainScreen.java

```diff
@@ -253,6 +253,8 @@ public void updateCustom(SolApplication solApplication, SolInputManager.InputPoi
             if (!isOn) {
                 is.setOperations(is.chooseMercenary);
                 inputMan.addScreen(solApplication, is);
+                
+                game.getHero().getTradeContainer().getMercs().seenAll();
```

Comment:
`seenAll`

Reply:
it makes sense. The player has "seen all" of the items. SO we can stop telling them about the new items (or mercenaries in this case).